sigma-ser/src/lib.rs

```diff
@@ -17,6 +17,8 @@
 #![deny(clippy::panic)]
 #![deny(clippy::wildcard_enum_match_arm)]
 
+/// ScoreX Serialization
+pub mod scorex_serialize;
```

Comment:
Could you please instead export module types from crate root? I mean:
```suggestion
mod scorex_serialize;
pub use scorex_serialize::{all needed types}
```
I'm trying to transition to this model since its more ergonomic (shorter imports and FQN).